apps/meteor/app/lib/server/functions/sendMessage.js

```diff
@@ -203,7 +203,7 @@ function cleanupMessageObject(message) {
 	['customClass'].forEach((field) => delete message[field]);
 }
 
-export const sendMessage = async function (user, message, room, upsert = false) {
+export const sendMessage = async function (user, message, room, upsert = false, previewUrls) {
```

Comment:
My idea here was not to use an empty array as the default value since it'd make no preview be rendered at all when the param isn't sent (I'm afraid this could cause some unexpected behavior somewhere else). Do you think we can proceed like this or should I add `= undefined` there maybe?

Reply:
pls add either `undefined` or `null` as the default value.. 😉

Reply:
tkss, done!

Reply:
Actually, adding `undefined` or `null` makes the ts files that use this method infer that this field is of type `undefined` or `null` :tasso:
I thought of converting this file to TS, but there are a lot of changes to be made :grimacing: Maybe we can keep no default value for now and convert this whole file to TS in another PR, wdyt? Or should we do this here?

Reply:
Done, I used `undefined` as the default value and defined the type using JSDoc. Thanks for the suggestion!